includes/discovery/sensors/signal/ceragon.inc.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+echo 'CERAGON-NET-MIB-FIBEAIR-4800 ';
+
+$divisor = 1;
+$type    = 'ceragon';
+
+$oids = snmp_walk($device, 'fibeAir4800OduAirRxPower', '-OsqnU', 'CERAGON-NET-MIB-FIBEAIR-4800');
```

Comment:
snmpwalk_cache_oid() or one of the other snmpwalk_* functions should be used instead of snmp_walk() as it formats data better.